tests/test_pdir_format.py

```diff
@@ -19,12 +19,13 @@ def test_pdir_module():
              '\x1b[0m\x1b[0;36m__package__\x1b[0m'),
             '\x1b[0;33mspecial attribute:\x1b[0m',
             '    \x1b[0;36m__doc__\x1b[0m', '\x1b[0;33mclass:\x1b[0m',
```

Comment:
The output of these tests is exactly the same by print. Here are some changes on color wrapper due to different ways the strings are formatted